.github/workflows/test.yml

```diff
@@ -21,6 +21,14 @@ jobs:
         - { os: windows-2016, ruby: mingw }
         - { os: windows-2019, ruby: mingw }
         - { os: windows-2019, ruby: mswin }
+        - { os: windows-2022, ruby: '2.0' }
+        - { os: windows-2022, ruby: 2.3   }
+        - { os: windows-2022, ruby: 2.4   }
+        - { os: windows-2022, ruby: '3.0' }
+        - { os: windows-2022, ruby: 3.1   }
+        - { os: windows-2022, ruby: mingw }
+        - { os: windows-2022, ruby: ucrt  }
+        - { os: windows-2022, ruby: head  }
```

Comment:
I think we should add `windows-2022` under `os:` so it's tested on all Ruby versions like 2016/2019.
Potentially we can drop windows-2016 at some point so we keep only 2 versions per OS.

Reply:
Updated.  I left out mswin on windows-2022.  See https://bugs.ruby-lang.org/issues/18362

Reply:
I still see those `include` in the diff, but they seem redundant now

Reply:
fixed